mindsdb/integrations/handlers/github_handler/github_tables.py

```diff
@@ -1058,4 +1058,129 @@ def get_columns(self) -> List[str]:
             "following",
             "created_at",
             "updated_at"
-        ]
\ No newline at end of file
+        ]
+        
+class GithubMilestonesTable(APITable):
+    """The GitHub Milestones Table implementation"""
+
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        """Pulls data from the GitHub "List repository milestones" API
+
+        Parameters
+        ----------
+        query : ast.Select
+           Given SQL SELECT query
+
+        Returns
+        -------
+        pd.DataFrame
+
+            GitHub milestones matching the query
+
+        Raises
+        ------
+        ValueError
+            If the query contains an unsupported condition
+        """
+
+        select_statement_parser = SELECTQueryParser(
+            query,
+            'milestones',
+            self.get_columns()
+        )
+
+        selected_columns, where_conditions, order_by_conditions, result_limit = select_statement_parser.parse_query()
+
+        total_results = result_limit if result_limit else 20
+
+        self.handler.connect()
+
+        github_milestones_df = pd.DataFrame(columns=self.get_columns())
+
+        start = 0
+
+        while True:
+            try:
+
+                for milestone in self.handler.connection.get_repo(self.handler.repository).get_milestones()[start: start + 10]:
+                    
+                    raw_data = milestone.raw_data
+                    github_milestones_df = pd.concat(
+                        [
+                            github_milestones_df,
+                            pd.DataFrame(
```

Comment:
Instead of concatenating the DataFrame in each iteration, it's better to append the new data to a list and then create a DataFrame once

Reply:
All the tables follow the same logic. I'll add this optimization in all tables in a different PR once this merged